src/mlpack/core/util/cli.cpp

```diff
@@ -86,165 +207,6 @@ CLI::~CLI()
   // may not have wanted it there (i.e. in Boost unit tests).
```

Comment:
One of this picky comments: Wasn't there a "rule" that words at the beginning of a sentences are capitalised?